templates/chain.html

```diff
@@ -285,6 +290,12 @@ <h4 style="margin-top:10px;">OPTIONAL:</h4>
                       <td><p id="dash-finish"></p></td>
                     </tr>
                   </table>
+                    <div class="op-control">
+                        <span title="Current Operation Status" class="op-control-text" id="op-control-state">Select or Add an Operation</span>
```

Comment:
subjective: remove the border around this (I think it'd look better without it) and remove the default text "SELECT OR ADD AN OPERATION" (not needed, I think it makes it busy for no reason).